packages/astro/src/internal/utils/versionSelector.ts

```diff
@@ -0,0 +1,7 @@
+export const versionSelector = (clerkJSVersion: string | undefined): string => {
+  if (clerkJSVersion) {
+    return clerkJSVersion;
+  }
+
+  return '5';
```

Comment:
❓ Consider moving this to a constant so it's easier to find and change in the future.